src/main/java/org/jitsi/jicofo/auth/ShibbolethHandler.java

```diff
@@ -242,8 +242,23 @@ private void doHandle(
                 "(function() {\n" +
                 " var sessionId = '" + sessionId + "';\n" +
                 " localStorage.setItem('sessionId', sessionId);\n" +
-                " console.info('sessionID :' + sessionId);\n";
-
+                " console.info('sessionID :' + sessionId);\n" +
+                " var displayName = '" + displayName + "';\n" +
+                " console.info('displayName :' + displayName);\n" +
+                " var settings = localStorage.getItem('features/base/settings');\n" +
+                " console.info('settings :' + settings);\n" +
+                " if (settings){\n" +
+                "     try {\n" +
+                "	        var settingsObj = JSON.parse(settings);\n" +
+                "	        if ( settingsObj != null && settingsObj.displayName != null && settingsObj.displayName == \"\" ) {\n" +
```

Comment:
do you think a truthy check for `settingsObj.displayName` would be good enough ?
`if (settingsObj != null && !settingsObj.displayName)`

Reply:
You are perfectly right Pawel, a truthy check is enough. 
Thanks.